src/Microsoft.TemplateEngine.Edge/EngineEnvironmentSettings.cs

```diff
@@ -16,20 +16,33 @@ namespace Microsoft.TemplateEngine.Edge
 {
     public sealed class EngineEnvironmentSettings : IEngineEnvironmentSettings
     {
+        /// <summary>
+        /// Instantiates the class.
+        /// </summary>
+        /// <param name="host"><see cref="ITemplateEngineHost"/> implementation for given host.</param>
+        /// <param name="virtualizeSettings">if true, settings will be stored in memory and will be disposed with instance.</param>
+        /// <param name="settingsLocation">the file path to store template engine settings, including global settings. To change host settings path only, use <paramref name="hostSettingsLocation"/>.</param>
+        /// <param name="hostSettingsLocation">the file path to store host specific settings. Use null for default location.
+        /// Note: this parameter changes only location of host specific settings. Global settings path remains unchanged.</param>
+        /// <param name="environment">the environment to be used.</param>
+        /// <param name="componentManager">the component manager to be used.</param>
+        /// <param name="pathInfo">the paths for the settings location.</param>
         public EngineEnvironmentSettings(
             ITemplateEngineHost host,
             bool virtualizeSettings = false,
             string? settingsLocation = null,
+            string? hostSettingsLocation = null,
```

Comment:
I'm not very happy about this change - should we make `DefaultPathInfo` public and override required properties in `Bootstrapper` instead?